src/navigators/RootStackNavigator/index.tsx

```diff
@@ -9,30 +9,48 @@ import { NAMESCREEN } from '../nameScreen';
 import { useSelector } from 'react-redux';
 import { ReduxStoreType } from '../../redux/store';
 import { LoginScreen } from '../../screens/LoginScreen';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
 
 export const RootStackNavigator: FC = () => {
   const Stack = createNativeStackNavigator<RootStackNavigationParamList>();
 
   const isSignIn = useSelector<ReduxStoreType>(state => {
-    return state.userToken;
+    return state.auth.userToken;
   });
   return (
-    <NavigationContainer>
-      <Stack.Navigator>
-        {isSignIn ? (
-          <React.Fragment>
+    <SafeAreaProvider>
+      <NavigationContainer>
+        <Stack.Navigator>
+          {isSignIn ? (
+            <React.Fragment>
```

Comment:
we don't need to have it here. The fragment we are using for 2 and more elements, here 1

Reply:
I agree

Reply:
yep. I see, sorry